Lib/test/test_array.py

```diff
@@ -1680,6 +1681,61 @@ def test_gh_128961(self):
         it.__setstate__(0)
         self.assertRaises(StopIteration, next, it)
 
+    @subTests(
+        ("dtype", "items"),
+        [
+            ("b", [0] * 64),
+            ("B", [1, 2, 3]),
```

Comment:
Any reason we need [0] * 64 and [1,2,3]? if not, we can just always use `list(range(64))` as the data.